client/components/userList.js

```diff
@@ -17,16 +13,19 @@ class UserList extends Component {
     componentDidMount() {
         fetch('http://localhost:9000/users')
             .then(response => response.json())
-            .then(data => this.setState({
-                users: data.user.map(user => {
-                    return {
-                        username: user.username,
-                        nativeLanguage: user.nativeLanguage,
-                        points: user.points,
-                        rank: user.rank
-                    }
+            .then(data => {
+                console.log(data)
+                this.setState({
+                    users: data.user.map(user => {
```

Comment:
@mwhitman189 I think it's better to do the mapping operation outside of set state.